front/src/components/login_forms.js

```diff
@@ -22,37 +22,42 @@ const buttonStyle = {
 };
 
 export const StudentLogin = () => {
+  const {loginUserStudent} = useContext(AuthContext);
   return (
     <Grid>
       <Paper elevation={0} style={paperStyle}>
         <Grid align="center">
           <Typography variant="h5">Rejoindre une session</Typography>
         </Grid>
         <Grid>
-          <TextField
-            id="standard-basic"
-            label="Room id"
-            placeholder="Entrer l'id de la room"
-            variant="standard"
-            fullWidth
-            style={buttonStyle}
-          />
-          <TextField
-            id="standard-basic"
-            label="Pseudo"
-            placeholder="Entrer un Pseudo"
-            variant="standard"
-            fullWidth
-            style={buttonStyle}
-          />
-          <Button
-            type="submit"
-            color="primary"
-            variant="contained"
-            fullWidth style={buttonStyle}
-          >
-            Rejoindre
-          </Button>
+          <form onSubmit={loginUserStudent}>
+            <TextField
+              id="standard-basic"
+              label="Room id"
+              placeholder="Entrer l'id de la room"
+              variant="standard"
+              fullWidth
+              name="classroom"
```

Comment:
The name should be more precise in my opinion
```
name="classroom_id"
```
or 
```
name="room_id"
```

Reply:
Done